cli/cmd/root.go

```diff
@@ -106,6 +106,7 @@ func NewCmdRoot() *cobra.Command {
 		NewPlayCmd(),
 		NewCartridgeCmd(),
 		NewCoredumpCmd(),
+		NewCleanCmd(),
```

Comment:
Insert the new command at the end of the list.

Reply:
fixed